spec/controllers/between_categories_transactions_controller_spec.rb

```diff
@@ -18,8 +18,10 @@
     context 'when valid' do
       let(:amount) { BigDecimal.new(Faker::Number.decimal(3, 2)) }
       let(:amount_from) { BigDecimal.new(10_000) }
-      let(:category_from) { create(:category, user: user, amount: amount_from) }
-      let(:category_to) { create(:category, user: user) }
+      let(:category_from) do
+        create(:main_category, user: user, amount: amount_from, categorizable: user)
```

Comment:
Isn't it useless to type `categorizable: user` here? 
Isn't `:main_category` already cared about it?